includes/CLI/Plugin_Check_Command.php

```diff
@@ -17,6 +19,8 @@
 
 /**
  * Plugin check command.
+ *
+ * @SuppressWarnings(PHPMD.ExcessiveClassComplexity)
```

Comment:
I don't think we should ignore this warning here. Either we decide we don't care about this, in which case we should ignore it in the PHPMD config file. Or we should address the problem that this class has. I would prefer to try the latter.

Can you try removing this and see whether you can find a solution? Potentially my other feedback about avoiding the duplicate logic between the 3 `get_*_options()` methods will already help with this.